Makefile

```diff
@@ -247,11 +247,19 @@ ide: zephyr
 .PHONY: ashell
 ashell: zephyr
 
-# Flash Arduino 101 x86 and arc images
+# Flash for Arduino 101 x86 and arc images, olimex_stm32_e407, 96b_carbon
```

Comment:
maybe replace with a simpler line like "Flash images", instead of listing every board in the same line, since we'll grow to support more boards, and put any board-specific comments below before each board if neccessary.